tests/Jellyfin.Model.Tests/Extensions/StringHelperTests.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using MediaBrowser.Model.Extensions;
+using Xunit;
+
+namespace Jellyfin.Model.Tests.Extensions
+{
+    public class StringHelperTests
+    {
+        [Theory]
+        [InlineData("", "")]
+        [InlineData("banana", "Banana")]
+        [InlineData("Banana", "Banana")]
+        [InlineData("ä", "Ä")]
+        public void StringHelper_ValidArgs_Success(string str, string result)
```

Comment:
I think you missed the parameter name suggestions in my last comment. These seem a bit more descriptive to me

```suggestion
        public void StringHelper_ValidArgs_Success(string input, string expectedResult)
```